quickstart/k8s-the-prod-way/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Kubernetes the Prod Way
+---
+
+**Kubernetes the Prod Way** is a tutorial, reference architecture, and collection of prod-first code
+examples that demonstrate industry best-practices for **using Kubernetes** in contexts where an
+**organization of people** must ship **production applications.**
+
+For example: in an organization, we typically expect identity (_e.g._, [AWS IAM][aws-iam], [GCP
+IAM][gcp-iam], [Azure AD][azure-ad]), compute (_e.g._, [EKS][eks], [GKE][gke], [AKS][aks]), storage
+(_e.g._, [Aurora][aurora], [Cloud SQL][cloud-sql], [CosmosDB][cosmos-db]), and networking to be
+provisioned and "owned" by separate people, and perhaps separate teams. But, when an app team
+deploys a service, we expect all of these components to work seamlessly together.
+
+**Kubernetes the Prod Way** will show you, using batteries-included examples, how to **provision**
+and **use** these technologies together in a way that maintains high release velocity, without
+sacrificing security, governance, or stability.
+
+Examples are provided for each of: **AWS, GCP, and Azure.**
+
+## Target Audience
+
+This tutorial is aimed at people who are planning to support production applications running on
+Kubernetes, particularly those looking for concrete guidance on how to set up infrastructure so that
+teams can operate quickly, effectively, and safely.
+
+## Contents
+
+The labs in Kubernetes the Prod Way are built using [Pulumi][pulumi], a tool that allows you to
+provision and configure cloud infrastructure, including [Amazon Web Services][aws] (AWS), [Microsoft
+Azure][azure], [Google Cloud Platform][gcp] (GCP), and Kubernetes.
+
+With that said, nearly all of the lessons learned could be applied using other tools as well, and
+there is very little that is specific to Pulumi.
```

Comment:
I'm not sure that's true and I wouldn't be so quick to walk back on Pulumi's value here. For instance, it is the reason we can use one set of tools and workflows across the cloud and Kubernetes resources.

Reply:
We can take it out but we'll need to add it back when we write posts for, _e.g._, the CNCF, or propose talks. This isn't just us being modest, this is us explaining lessons learned _through_ Pulumi.

Reply:
This is a page for pulumi.io. We shouldn't shy away from emphasizing what Pulumi does well.